### DISCUSSION TITLE
Improve Admin tools -> Players detail pages

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#760] [Suggestions]
Currently, the display for all of the above and more is more or less the same, a log with pagination that goes on and on and on. 
![image](https://github.com/gameserverapp/Platform/assets/44717221/1889d936-6559-4ac0-b9a3-74088db790e3)
![image](https://github.com/gameserverapp/Platform/assets/44717221/46e2d30a-ca03-4edd-85a3-dba01fe93b0b)
![image](https://github.com/gameserverapp/Platform/assets/44717221/b1443fc7-3b06-4bc6-b34b-d82606804701)
![image](https://github.com/gameserverapp/Platform/assets/44717221/07659ac9-06c1-4963-8480-d080fe0380ca)
![image](https://github.com/gameserverapp/Platform/assets/44717221/9b2355b4-2c54-47f5-b5f1-d297a1728fa3)

I am requesting an upgrade to these pages that allow for filtering based on the appropriate task, pack, etc. that delivers the log entry into the players' history, similar to how we have such a filtering system with the task run logs, pictured below. Maybe some additional filtering functions, like date ranges, would help with navigating more extensive logs. 
![image](https://github.com/gameserverapp/Platform/assets/44717221/3e4f86cc-8c42-4472-8e8f-a2a1128809df)

**Comment by mvaessen** [comment]
Thanks for the great suggestion! Almost all of these suggestions are either already added or will be published tomorrow.
Do note that this is currently only available on Admin Tools V2
